guides/doc-Provisioning_Guide/topics/Virt-RHV.adoc

```diff
@@ -101,54 +123,56 @@ To create a Red Hat Virtualization connection, enter the `hammer compute-resourc
 
 Optionally, to evaluate the new engine API, change `false` to `true` for the `--use-v4` option.
 
+ifeval::["{build}" == "satellite"]
 [WARNING]
 ====
 The items listed in this step are provided as Technology Previews. For further information about the scope of Technology Preview status, and associated support implications, see https://access.redhat.com/support/offerings/techpreview/[Technology Preview Features Support Scope].
 ====
+endif::[]
 
 [[Provisioning_Virtual_Machines_in_Red_Hat_Virtualization-Adding_Red_Hat_Virtualization_Images_on_the_Satellite_Server]]
-=== Adding Red Hat Virtualization Images to {ProjectServer}
+=== Adding {OVirt} Images to {ProjectServer}
 
-Red Hat Virtualization uses templates as images for creating virtual machines. If you use image-based provisioning to create hosts, you must add Red Hat Virtualization template details to your {ProjectServer}. This includes access details and the template name.
+{OVirt} uses templates as images for creating virtual machines. If you use image-based provisioning to create hosts, you must add {OVirt} template details to your {ProjectServer}. This includes access details and the template name.
 
 .Procedure
 
-To add Red Hat Virtualization images on {ProjectServer}, complete the following steps:
+To add {OVirt} images on {ProjectServer}, complete the following steps:
 
-. In the {Project} web UI, navigate to *Infrastructure* > *Compute Resources*, and in the Compute Resources window, click the name of your Red Hat Virtualization connection.
+. In the {Project} web UI, navigate to *Infrastructure* > *Compute Resources*, and in the Compute Resources window, click the name of your {OVirt} connection.
 . Click the *Image* tab, and then click *New Image*.
 . In the *Name* field, enter a name for the image.
 . From the *Operatingsystem* list, select the image's base operating system.
 . From the *Architecture* list, select the operating system architecture.
 . In the *Username* field, enter the SSH user name for image access. This is normally the `root` user.
 . In the *Password* field, enter the SSH password for image access.
-. From the *Image* list, select the name of the image on Red Hat Virtualization.
+. From the *Image* list, select the name of the image on {OVirt}.
 . Click *Submit* to save the image details.
 
 .For CLI Users
 
-Create the image with the `hammer compute-resource image create` command. Use the `--uuid` option to store the template UUID on the Red Hat Virtualization server.
+Create the image with the `hammer compute-resource image create` command. Use the `--uuid` option to store the template UUID on the {OVirt} server.
 
 [options="nowrap" subs="+quotes"]
 ----
-# hammer compute-resource image create --name "_Test_RHV_Image_" \
+# hammer compute-resource image create --name "_Test_OVirt_Image_" \
```

Comment:
Here too, just replace the RHEV with {OVirt} and it should render accordingly?

Reply:
Deal.